hr_planning_resources/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,20 @@
+from odoo import api, models
+
+
+class HelpdeskTicket(models.Model):
+    _name = "helpdesk.ticket"
+    _inherit = ["helpdesk.ticket", "hr.task.mixin"]
+
+    def _compute_hr_task_count(self):
+        for record in self:
+            ticket_count = self.env["hr.task"].search_count(
+                [("ticket_id", "=", record.id)]
+            )
+            record.hr_task_count = ticket_count
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        if self.env.context.get("default_user_id", False):
```

Comment:
This context key should already put this value (on the form view) as a default. And maybe the user change it (and you re-force it?)